cdap-docs/application-templates/source/etl/templates/sources/real-time/twitter.rst

```diff
@@ -6,9 +6,29 @@
 Sources: Real-time: Twitter 
 ===============================
 
-.. rubric:: Description: Twitter Realtime Source
+.. rubric:: Description
 
-Output records contain the fields *id* (long), *message* (string), *lang* (nullable
-string), *time* (nullable long), *favCount* (int), *rtCount* (int), *source* (nullable
+Samples tweets in realtime. Output records contain the fields *id* (long), *message* (string),
+*lang* (nullable string), *time* (nullable long), *favCount* (int), *rtCount* (int), *source* (nullable
 string), *geoLat* (nullable double), *geoLong* (nullable double), and *isRetweet*
```

Comment:
Since Twitter uses a long-lat format, what about keeping these fields in that order?
```
*geoLong* (nullable double), *geoLat* (nullable double),...